components/raftstore/src/store/fsm/peer.rs

```diff
@@ -270,7 +270,10 @@ impl BatchRaftCmdRequestBuilder {
         // No batch request whose size exceed 20% of raft_entry_max_size,
         // so total size of request in batch_raft_request would not exceed
         // (40% + 20%) of raft_entry_max_size
-        if req.has_admin_request() || f64::from(req_size) > self.raft_entry_max_size * 0.2 {
+        if req.has_admin_request()
+            || req.has_status_request()
```

Comment:
What if another new type is added to `RaftCmdRequest`? How about `!req.has_requests()`?

Reply:
Good idea.